test/test_temporal_average.cpp

```diff
@@ -15,9 +22,12 @@ using namespace std;
 
 int main(int argc, char **argv)
 {
+    teca_mpi_manager mpi_man(argc, argv);
+    int rank = mpi_man.get_comm_rank();
+
     teca_system_interface::set_stack_trace_on_error();
 
-    if (argc < 7)
+    if (argc < 5)
```

Comment:
this looks wrong to me. the argument parsing code below will crash if `argc < 7`. at least one array is expected